### PR TITLE
TensorBoard Modifications for Word2Vec Example

### PR DESCRIPTION
**Pull request by ozgyal**
TensorBoard modifications are added to Word2Vec example for visualizing the loss graphic and embeddings with proper words in TensorBoard.

### CODE REVIEW

**Comment by ozgyal** [comment]
I signed CLA.
Thanks.

**Comment by nealwu** [comment]
I'm not the right person to review this since I haven't worked much with this code. Going to unassign myself.

**Comment by drpngx** [comment]
Re-routing, @jart feel free to bounce.

**Comment by drpngx** [comment]
@jart WDYT?

File: tensorflow/examples/tutorials/word2vec/word2vec_basic.py [hunk 1]
```diff
@@ -197,13 +208,27 @@ def generate_batch(batch_size, num_skips, skip_window):
...
   # Add variable initializer.
   init = tf.global_variables_initializer()
 
+  # Create a saver.
+  saver = tf.train.Saver()
+
 # Step 5: Begin training.
 num_steps = 100001
 
+log_path = 'your_full_path/log'
```
Comment (jart) [inline tensorflow/examples/tutorials/word2vec/word2vec_basic.py]: Would it be possible to have a flag for the log directory? Sort of like [mnist_with_summaries.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist_with_summaries.py)?

**Review by jart — DISMISSED** [review]
Thank you for the contribution! One small but important request.

File: tensorflow/examples/tutorials/word2vec/word2vec_basic.py (hunk 1 above)
Comment (ozgyal) [reply tensorflow/examples/tutorials/word2vec/word2vec_basic.py]: I have made the changes, thanks for the comment.

Comment (drpngx) [reply tensorflow/examples/tutorials/word2vec/word2vec_basic.py]: I can't see the changes, could you check?

Comment (ozgyal) [reply tensorflow/examples/tutorials/word2vec/word2vec_basic.py]: I've checked but I couldn't find the problem. The last commit for log directory changes: https://github.com/tensorflow/tensorflow/pull/14908/commits/7d18efae4d355713f9dc600814dad73d2aeccef5

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by drpngx** [comment]
Looks like @ozgyal made the last requested changes. Setting as good to go.

**Comment by drpngx** [comment]
Jenkins, test this please.

**Review by drpngx — APPROVED** [review]
Verified that `log_dir` was added as requested.
